### ISSUE TITLE
Attach weiss gripper model to robot urdf

### ISSUE THREAD

**Issue by radhen** [nalt/wsg50-ros-pkg#7]
Is there a way I can attach the Weiss gripper model to the robot urdf in RViz and read the finger tip link transform in real time? In short I want to know where the finger tip of the gripper is in real time w.r.t to the base of the robot.

**Comment by radhen** [comment]
I think the answer lies within the wsg_50_simulation folder. I see it launches the gripper model and necessary joint state publishers which should be reading the current position of the gripper and publishing it back into RViz. I launched the wsg_50.launch 

`roslaunch wsg_50_simulation wsg_50.launch`

but no success. It breaks at certain point. Currently debugging into this to see what exactly is going on. If someone has tried this in the past and got it working then please comment below. It will save a good amount of my time.

**Comment by radhen** [comment]
So I looked at the wsg_50.launch file properly. Unfortunately that is not what I am trying to achieve. If I understand it properly the file launches fake joint controllers or a simulated gripper in RViz. I wish to read the gripper status in real time. 

Fortunately I managed to get a work around this. Currently I am using the [send_urdf_fragment.py](https://github.com/RethinkRobotics/intera_sdk/blob/master/intera_interface/scripts/send_urdf_fragment.py) file to attach the gripper urdf fragment to the robot urdf and publish it in RViz. I set the gripper parameters (finger_length, finger_tip etc) in wsg_50.urdf.xacro file according to my needs and run the python file and I can now visualize the gripper attached to the robot urdf in RViz.
